lib/mix/tasks/ecto.gen.migration.ex

```diff
@@ -94,11 +94,22 @@ defmodule Mix.Tasks.Ecto.Gen.Migration do
 
   embed_template :migration, """
   defmodule <%= inspect @mod %> do
-    use Ecto.Migration
+    use <%= libraryName() %>.Migration
 
     def change do
   <%= @change %>
     end
   end
   """
+  
+  defp libraryName() do
+    case Application.get_env(:ecto_sql, :migration_library) do
```

Comment:
My suggestion is to call it `migration_module` and have it be the whole module and not only the prefix. It should also be an atom, so you have to put it on  the template  as `<%= inspect migration_module() %>`.